libeos-parental-controls/app-filter.c

```diff
@@ -872,6 +907,29 @@ epc_set_app_filter (GDBusConnection  *connection,
       return FALSE;
     }
 
+  allow_user_installation_result_variant =
+      g_dbus_connection_call_sync (connection,
```

Comment:
Might be worth factoring out a `Properties.Set` helper at this point?

Reply:
Yes and no. Yes, because that would make the current code nicer. No, because it would reduce the incentive to fix https://gitlab.freedesktop.org/accountsservice/accountsservice/issues/66 and condense all the D-Bus calls into a single call.

For the moment, I’d prefer to hope that https://gitlab.freedesktop.org/accountsservice/accountsservice/issues/66 gets implemented (probably by me). I may reassess this hopefulness next time I have to add a property to `epc_set_app_filter()` though.